st2common/bin/st2-self-check

```diff
@@ -59,17 +59,23 @@ elif [ -e "${ST2_CLI_CONFIG_FILE}" ]; then
   fi
 fi
 
+pushd ~
```

Comment:
@kami: https://github.com/StackStorm/st2/pull/3084#discussion_r90782303 and https://github.com/StackStorm/st2/pull/3084/commits/b6a6de7ef7e05c0a56fa204596b938d1da6fd111#diff-94c8a3b53a504995343d0f973102c9f1R64 😆